### PR TITLE
Hello Microverse project submission 

### PR DESCRIPTION
**Pull request by ShaheerCH**
- Added HTML-CSS Linters
- Added Index.html 
- Added *h1* tags in Header
- Added Stylesheet.css
- Added Font size and color to heading
- Added text.md to .gitignore

### CODE REVIEW

**Comment by ShaheerCH** [comment]
Add-Linters passed all the checks

File: READMEE.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+1. In the first commit of your feature branch create a `.github/workflows` folder and add a copy of [`.github/workflows/linters.yml`](.github/workflows/linters.yml) to that folder.
+   - **Remember** to use the file linked above
+   - **Remember** that `.github` folder starts with a dot.
+2. **Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.**
+   - If you think that change is necessary - open a [Pull Request in this repository](../README.md#contributing) and let your code reviewer know about it.
+3. When you open your first pull request you should see the result of the GitHub actions checks:
+
+![gh actions checks](../assets/images/gh-actions-html-css-checks.png)
+
+Click on the `Details` link of each action to see the full output and the errors that need to be fixed:
```
Comment (phelian23) [inline READMEE.md]: - [x] Good job adding a descriptive READMEE.md file, however, the name should be README.md, not READMEE.md. Kindly copy the content into the REAME.md file and delete the READMEE.md file

**Review by phelian23 — CHANGES_REQUESTED** [review]
Hi @ShaheerCH, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ Linters are passing
✅ Descriptive pull request
✅ Good commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ShaheerCH** [comment]
Edited the read.me file

**Comment by ShaheerCH** [comment]
Add-Linters passed all the checks

**Review by brytebee — APPROVED** [review]
Hi, @ShaheerCH👋,

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**